zadani/soft/style.css

```diff
@@ -11,33 +11,289 @@
 body {
     margin: 0;
     padding: 0;
-    background-color: #f5f5f5;
+    background-color: white;
     font-family: 'Merriweather', serif;
     font-size: 16px;
     line-height: 1.6;
 }
 
 
 /* HLAVICKA */
-
+.zahlavi {
+    background-image: URL("../../images/island-hlavicka.jpg");
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    color: white;
+    text-align: center;
+}
 .titulek {
+    font-size:3rem;
     font-family: 'Raleway', serif;
+    padding-top:20px;
+    padding-bottom:0px;
+
```

Comment:
klidně bych si u titulku vynulovala margin (margin: 0), ať tam nemáš tu mezeru na počátku. Ale to je jen drobnost. :)